src/mod.ts

```diff
@@ -1,5 +1,6 @@
 export const BUILDER_TAG = "deno_slack_builder@0.0.10";
 export const RUNTIME_TAG = "deno_slack_runtime@0.0.6";
+export const HOOKS_TAG = "deno_slack_hooks@0.0.1";
```

Comment:
Why is this `0.0.1`? It looks like the latest `deno_slack_hooks` is now `0.0.6`. Is it possible to use this packages tagged version instead of hardcoding the hooks version?